<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>World Clock</string>
       </attribute>
       <widget class="QLabel" name="clock_label_iran">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>160</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tehran</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_germany">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Berlin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_usa">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>380</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NewYork</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_iran_time">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_germany_time">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>260</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_usa_time">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>380</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_gmt">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GMT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_gmt_time">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="clock_label_seconds">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>40</y>
          <width>211</width>
          <height>401</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>120</pointsize>
         </font>
        </property>
        <property name="text">
         <string>00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Alarm</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>711</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="alarm_grid"/>
       </widget>
       <widget class="QLabel" name="label_alam">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add Alarm:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="alarm_btn_add">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>80</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="alarm_time_edit">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>80</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="alarm_txtbox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>191</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>StopWatch</string>
       </attribute>
       <widget class="QLabel" name="label_stopwatch">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>311</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(145, 65, 172);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="stopwatch_start_btn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>240</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopwatch_stop_btn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>240</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopwatch_reset_btn">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>240</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Timer</string>
       </attribute>
       <widget class="QLineEdit" name="timer_hour_txtbox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="timer_minute_txtbox">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="text">
         <string>15</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="timer_second_txtbox">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>80</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Seven Segment</family>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="timer_start_btn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="timer_stop_btn">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>310</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="timer_reset_btn">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>310</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="iconSize">
         <size>
          <width>19</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
